src/fidesops/util/cache.py

```diff
@@ -147,3 +147,7 @@ def get_all_cache_keys_for_privacy_request(privacy_request_id: str) -> List[Any]
     return cache.keys(f"{privacy_request_id}-*") + cache.keys(
         f"id-{privacy_request_id}-*"
     )
+
+
+def get_async_task_tracking_cache_key(privacy_request_id: str) -> str:
+    return f"id-{privacy_request_id}-async-execution"
```

Comment:
In our current design I believe there could be multiple celery tasks for the same privacy request. For example, if you "pause" a request, behind the scenes, this stops executing remaining tasks and exits.  Resuming a privacy request would create another task for the same privacy request id.

I'm thinking we might need to differentiate here?

Reply:
I may be wrong in my understanding here — does the initial task finish when the privacy request execution pauses? If so, we should be OK overwriting the privacy request's `task_id` on resume